Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs

```diff
@@ -309,7 +315,7 @@ private Operation CreatePutByIdOperation(OpenApiMetadataPathsResource openApiMet
                     new Response
                     {
                         description =
-                            "Method Is Not Allowed. When the Snapshot-Identifier header is present the method is not allowed."
+                            "Method Is Not Allowed. When the Use-Snapshot header is set to true, the method is not allowed."
```

Comment:
Just nothing that this is not consistent with the usage of the enumeration values, but it is correct for where I think we should land.